ruby/deepgram-audio-summary/index.rb

```diff
@@ -0,0 +1,42 @@
+require "uri"
+require "net/http"
+require "openssl"
+require "json"
+
+
+def main(req, res)
+  fileUrl = nil
+  begin
+    payload = JSON.parse(req.payload)
+    fileUrl = payload["fileUrl"]
+  rescue Exception => err
+      puts err
+      raise "Payload is invalid."
+  end
+  
+  url = URI("https://api.deepgram.com/v1/listen?summarize=true&punctuate=true")
+  http = Net::HTTP.new(url.host, url.port)
+  http.use_ssl = true
+  http.verify_mode = OpenSSL::SSL::VERIFY_NONE
+
+  request = Net::HTTP::Post.new(url)
+  request["content-type"] = "application/json"
+  request["Authorization"] = "Token #{req.env["DEEPGRAM_API_KEY"]}"
+  request.body = "{\"url\":\"#{fileUrl}\"}"
+
+  begin
+    response = http.request(request)
+  rescue Exception => err
+    res.json({
+      success: false,
+      deepgramData: "Please provide a valid file URL."
+    })
+  end
+
+  parsed_response = JSON.parse(response.read_body)
+  summaries = parsed_response["results"]["channels"][0]["alternatives"][0]["summaries"]
```

Comment:
Let's return the full data from deepgram rather than just `parsed_response["results"]["channels"][0]["alternatives"][0]["summaries"]`

Reply:
full data meaning the  `parsed_response` or `parsed_response["results"]` ?

Reply:
@Azanul Just results :)

The rest of the response model contains metadata that we don't want to expose.

Reply:
```js
"metadata": {
    "transaction_key": "string",
    "request_id": "uuid",
    "sha256": "string",
    "created": "string",
    "duration": 0,
    "channels": 0,
    "models": [
      "string"
    ],
  },
```

This stuff is not meaningful to the end user.

Reply:
Noted.